iaso/api/entity.py

```diff
@@ -165,9 +166,27 @@ def get_queryset(self):
         if form_name:
             queryset = queryset.filter(attributes__form__name__icontains=form_name)
         if search:
-            queryset = queryset.filter(
-                Q(name__icontains=search) | Q(uuid__icontains=search) | Q(attributes__json__icontains=search)
-            )
+            if search.startswith("ids:"):
+                ids_str = search.replace("ids:", "")
+                try:
+                    ids = re.findall("[A-Za-z0-9_-]+", ids_str)
+                    queryset = queryset.filter(id__in=ids)
+                except:
+                    queryset = queryset.filter(id__in=[])
+                    print("Failed parsing ids in search", search)
```

Comment:
I would raise an explicit error in case of a parsing error:

```suggestion
                ids = re.findall("\d+", search)
                if not ids:
                    raise ValidationError(f"Failed parsing ids in search '{search}'")
                queryset = queryset.filter(id__in=ids)
```

Reply:
If you accept this suggestion, don't forget the import:

```from rest_framework.exceptions import ValidationError```